crates/exex/types/src/finished_height.rs

```diff
@@ -1,9 +1,10 @@
 use alloy_primitives::BlockNumber;
 
 /// The finished height of all `ExEx`'s.
-#[derive(Debug, Clone, Copy)]
+#[derive(Debug, Clone, Copy, Default)]
 pub enum FinishedExExHeight {
     /// No `ExEx`'s are installed, so there is no finished height.
+    #[default]
```

Comment:
I don't think we should rely on Default in this case, I would prefer to have an explicit enum variant in every scenario

Reply:
Understood, just reverted